### ISSUE TITLE
No "wrong password" warning

### ISSUE THREAD

**Issue by tjvr** [github/hub#1122]
Hi! I'm using OS X 10.11.3. I installed hub from homebrew.

hub kept asking for my password:

```
Desktop$ hub clone tjvr/scratchblocks
github.com username: tjvr
github.com password for tjvr (never stored): 
```

I checked and `.config/hub` wasn't created either.

This was terribly confusing, until I realised I was entering the *wrong password*, but hub wasn't complaining about this (I guess because the repository is public, so the clone still succeeds?)

```
$ hub clone tjvr/scratchblocks
github.com username: tjvr
github.com password for tjvr (never stored): 
Cloning into 'scratchblocks'...
remote: Counting objects: 1207, done.
remote: Compressing objects: 100% (75/75), done.
remote: Total 1207 (delta 35), reused 0 (delta 0), pack-reused 1130
Receiving objects: 100% (1207/1207), 782.43 KiB | 0 bytes/s, done.
Resolving deltas: 100% (775/775), done.
Checking connectivity... done.
```

A "wrong password" message would be appreciated :-)

**Comment by mislav** [comment]
Thanks for reporting. Does this happen with the latest version of hub as well?

**Comment by mislav** [comment]
@tjvr Is this still a problem with you in the latest version?

**Comment by tjvr** [comment]
I still see this on hub version 2.2.3. :)

**Comment by mislav** [comment]
Fix now included in the 2.2.4 release.
